docs/internals/job_scheduling.zh.md

```diff
@@ -22,81 +22,52 @@ specific language governing permissions and limitations
 
 <div style="text-align: center;">
 <img src="{{ site.baseurl }}/fig/job_and_execution_graph.svg" alt="JobGraph and ExecutionGraph" height="400px" style="text-align: center;"/>
 </div>
 
-Each ExecutionGraph has a job status associated with it.
-This job status indicates the current state of the job execution.
+每个 ExecutionGraph 都有一个与之相关的作业状态信息，用来描述当前的作业执行状态。
 
-A Flink job is first in the *created* state, then switches to *running* and upon completion of all work it switches to *finished*.
-In case of failures, a job switches first to *failing* where it cancels all running tasks.
-If all job vertices have reached a final state and the job is not restartable, then the job transitions to *failed*.
-If the job can be restarted, then it will enter the *restarting* state.
-Once the job has been completely restarted, it will reach the *created* state.
+Flink 作业刚开始会处于 *created* 状态，然后切换到 *running* 状态，当所有任务都执行完之后会切换到 *finished* 状态。如果遇到失败的话，作业首先切换到 *failing* 状态以便取消所有正在运行的 task。如果所有 job 节点都到达最终状态并且 job 无法重启， 那么 job 进入 *failed* 状态。如果作业可以重启，那么就会进入到 *restarting* 状态，当作业彻底重启之后会进入到 *created* 状态。
 
-In case that the user cancels the job, it will go into the *cancelling* state.
-This also entails the cancellation of all currently running tasks.
-Once all running tasks have reached a final state, the job transitions to the state *cancelled*.
+如果用户取消了 job 话，它会进入到 *cancelling* 状态，并取消所有正在运行的 task。当所有正在运行的 task 进入到最终状态的时候，job 进入 *cancelled* 状态。
 
-Unlike the states *finished*, *canceled* and *failed* which denote a globally terminal state and, thus, trigger the clean up of the job, the *suspended* state is only locally terminal.
-Locally terminal means that the execution of the job has been terminated on the respective JobManager but another JobManager of the Flink cluster can retrieve the job from the persistent HA store and restart it.
-Consequently, a job which reaches the *suspended* state won't be completely cleaned up.
+跟 *finished* 不同的是，*canceled* 和 *failed* 会导致全局的终结状态，并且触发作业的清理，*suspended* 状态只是一个局部的终结。
+局部的终结意味着作业有一部分 JobManager 被终结，但是另外一部分 JobManager 依然可以从高可用存储里获取作业并重启。因此一个处于 *suspended* 状态的作业不会被彻底清理掉。
```

Comment:
```suggestion
局部的终结意味着作业的执行已经被对应的 JobManager 终结，但是集群中另外的 JobManager 依然可以从高可用存储里获取作业信息并重启。因此一个处于 *suspended* 状态的作业不会被彻底清理掉。
```

Reply:
@wuchong Thanks for your refined translation. It's more reasonable. Does this means my first pull request will merge to Flink codebase ? 
ps: How to move my jira task issue (https://issues.apache.org/jira/browse/FLINK-12360) to subtask issue of "Support Chinese Documents for Apache Flink"  (https://issues.apache.org/jira/browse/FLINK-11529) ?

Reply:
Yes, I will merge you commit into Flink master codebase. 

Thanks for the reminder of it. I have converted it into a subtask of FLINK-11529.